### ISSUE TITLE
[Bug]: WebKit TypeError: undefined is not an object (evaluating 'binding.__installed')

### ISSUE THREAD

**Issue by Nubebuster** [microsoft/playwright#30414]
### Version

1.43.1

### Steps to reproduce

https://github.com/Nubebuster/playwright-report

cd new-project
npm run test:webkit

### Expected behavior

The page loads and is visible.

### Actual behavior

Page does not load. Console shows the following error:

[Error] TypeError: undefined is not an object (evaluating 'binding.__installed')
	addPageBinding (Inspector Bootstrap Script:8)
	Global Code (Inspector Bootstrap Script:54)

### Additional context

Other browsers run fine

### Environment

```shell
System:
    OS: Linux 6.5 Ubuntu 22.04.4 LTS 22.04.4 LTS (Jammy Jellyfish)
    CPU: (8) x64 Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
    Memory: 20.66 GB / 31.28 GB
    Container: Yes
  Binaries:
    Node: 18.19.0 - ~/.nvm/versions/node/v18.19.0/bin/node
    Yarn: 1.22.19 - /usr/bin/yarn
    npm: 10.2.3 - ~/.nvm/versions/node/v18.19.0/bin/npm
    pnpm: 8.6.3 - /usr/local/bin/pnpm
  Languages:
    Bash: 5.1.16 - /usr/bin/bash
  npmPackages:
    @playwright/test: ^1.43.1 => 1.43.1
```

**Comment by yury-s** [comment]
> npm run test:webkit

I'm getting the following error:
```
Error: Process from config.webServer was not able to start. Exit code: 127
```

**Comment by sunilsurana** [comment]
Hi,

We are also seeing similar error but intermittently. Although getTestEnvironmentInfo function is present it still says function not defined and strange things is we are not able to repro and happens randomly in some CI pipelnes

![image](https://github.com/microsoft/playwright/assets/7621111/604d3a4c-3404-4309-8788-42f7ecf7f0ee)

**Comment by yury-s** [comment]
Cannot reproduce it. Works with no issues for me in our 1.43.1-jammy docker image:

```
$ docker run -it --rm --ipc=host mcr.microsoft.com/playwright:v1.43.1-jammy /bin/bash
root@20b29d5e75fd:/# cd 
root@20b29d5e75fd:~# git clone https://github.com/Nubebuster/playwright-report.git
Cloning into 'playwright-report'...
remote: Enumerating objects: 40, done.
remote: Counting objects: 100% (40/40), done.
remote: Compressing objects: 100% (30/30), done.
remote: Total 40 (delta 5), reused 40 (delta 5), pack-reused 0
Receiving objects: 100% (40/40), 38.31 KiB | 726.00 KiB/s, done.
Resolving deltas: 100% (5/5), done.
root@20b29d5e75fd:~# cd playwright-report/new-project/
root@20b29d5e75fd:~/playwright-report/new-project# npm i

added 5 packages, and audited 6 packages in 601ms

found 0 vulnerabilities
npm notice 
npm notice New patch version of npm available! 10.5.0 -> 10.5.2
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.5.2
npm notice Run npm install -g npm@10.5.2 to update!
npm notice 
root@20b29d5e75fd:~/playwright-report/new-project# xvfb-run npx playwright test --headed

Running 3 tests using 3 workers
  3 passed (4.0s)

To open last HTML report run:

  npx playwright show-report

root@20b29d5e75fd:~/playwright-report/new-project# 
```

**Comment by gskierk** [comment]
@yury-s I have reproduced it with Playwright v1.46.1. MacOS. Webkit.

1. Create index.html including embedding to YouTube video only.
2. Start the test that just goes to the local HTML file with Desktop Webkit preset.
3. You have this error (TypeError: undefined is not an object (evaluating 'binding.__installed')) in the browser console.

```html
<iframe width="560" height="315" src="https://www.youtube.com/embed/8ssf762p3dY?si=NcP3v_UyQSgMHGQj" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
